test-cypress/integration/explore/visual.graph.spec.js

```diff
@@ -382,393 +228,352 @@ describe('Visual graph screen validation', function () {
+        getShowPreferredTypesOnlyCheckbox().check()
+            .should('be.checked');
+        getIgnoredTypesField().type('vin:PinotNoir')
+            .should('have.value', 'vin:PinotNoir');
+
+        // Go to predicates tab
+        openPredicatesTab();
+        // No pre-added preferred/ignored predicates
+        getPreferredPredicatesField().type('vin:hasSugar')
+            .should('have.value', 'vin:hasSugar');
+        getShowPreferredPredicatesOnlyCheckbox().check()
+            .should('be.checked');
+        getIgnoredPredicatesField().type('vin:hasSugar')
+            .should('have.value', 'vin:hasSugar');
+
+        saveSettings();
+
+        // Reset settings and verify everything is reverted to its default
+        openVisualGraphSettings();
+        resetSettings();
+        saveSettings();
+
+        openVisualGraphSettings();
+        // Verify that the default settings are as follows:
+        // Maximum links to show: 20
+        getLinksNumberField().should('have.value', '20');
+        // Preferred lang: en
+        cy.get('.preferred-languages .tag-item').should('have.length', 0);
+        // Include inferred: false
+        getIncludeInferredStatementsCheckbox().should('not.be.checked')
+            .and('not.be.disabled');
+        // Expand results over owl:sameAs: false
+        getSameAsCheckbox().should('not.be.checked')
+            .and('be.disabled');
+        // Show predicate labels: true
+        getShowPredicateLabelsCheckbox().should('be.checked')
+            .and('not.be.disabled');
+
+        // No pre-added preferred/ignored types
+        getPreferredTypesField().should('be.empty');
+        getShowPreferredTypesOnlyCheckbox().should(('not.be.checked'))
+            .and('not.be.disabled');
+        getIgnoredTypesField().should('be.empty');
+
+        // Go to predicates tab
+        openPredicatesTab();
+        // No pre-added preferred/ignored predicates
+        getPreferredPredicatesField().should('be.empty');
+        getShowPreferredPredicatesOnlyCheckbox().should(('not.be.checked'))
+            .and('not.be.disabled');
+        getIgnoredPredicatesField().should('be.empty');
     });
+
+    // Visual graph home view access
+
+    function getSearchField() {
+        return cy.get('.search-rdf-resources input:visible');
+    }
+
+    function typeInSearchField(resource) {
+        getSearchField().type(resource).trigger('change').wait(200).type('{enter}');
+    }
+
+    function searchForResource(resource) {
+        typeInSearchField(resource);
+        // Verify redirection to existing visual graph
+        cy.get('.nodes-container').should('be.visible');
+    }
+
+    function getTargetNodeElement() {
+        return cy.get(`[id="http://www.w3.org/TR/2003/PR-owl-guide-20031209/wine#${VALID_RESOURCE}"] circle`);
+    }
+
+    function getTargetNode() {
+        // timeout is needed because mouseover event will result in pop-up of menu icons only if nodes are not moving
```

Comment:
Timeout or wait? And still I don't know why is it needed when there is not command